packages/openapi-generator/src/wrapper-files/api-file.spec.ts

```diff
@@ -15,6 +15,16 @@ describe('api-file', () => {
               name: 'limit',
               type: 'number'
             },
+            {
```

Comment:
I think this test is overkill, because you are not testing anything that the api-file module does. If you remember from our discussion on our [test strategy](https://github.com/SAP/cloud-sdk-js/blob/main/knowledge-base/adr/0010-test-strategy.md#unit-tests), we agreed that everything should be tested **once**. This is already tested in the parser, so I would rather like to remove this.